packages/nextjs/src/config/loaders/wrappingLoader.ts

```diff
@@ -26,7 +26,6 @@ const pageWrapperTemplateCode = fs.readFileSync(pageWrapperTemplatePath, { encod
 const middlewareWrapperTemplatePath = path.resolve(__dirname, '..', 'templates', 'middlewareWrapperTemplate.js');
 const middlewareWrapperTemplateCode = fs.readFileSync(middlewareWrapperTemplatePath, { encoding: 'utf8' });
 
-const requestAsyncStorageShimPath = path.resolve(__dirname, '..', 'templates', 'requestAsyncStorageShim.js');
```

Comment:
seeing that we have a few of these below, should we all replace them with an `@sentry/nextjs/*` path?

Reply:
So this particular thing we're changing in this PR is actually something we let webpack import for us. All of the other paths in this file are actual templates we just call `fs.readFile` on, so no need to have some sort of special resolution as of now.